packages/react/slider/src/Slider.tsx

```diff
@@ -643,6 +632,7 @@ const SliderThumbImpl = React.forwardRef((props, forwardedRef) => {
       }}
     >
       <Primitive
+        data-radix-slider-thumb=""
```

Comment:
inlined for now as we rely on it in the `isThumb` function, but we will most likely do this in a different way in another PR.